libs/blocks/section-metadata/section-metadata.css

```diff
@@ -201,7 +201,7 @@
 
 .section.sticky-bottom.promo-sticky-section {
   background: none;
-  z-index: 4;
+  z-index: 40000;
```

Comment:
I am guessing the other martech element has a high z-index like this as well?

Reply:
Yes, the dynamic chat is at 10k.